.storybook/themes/news.scss

```diff
@@ -2618,6 +2618,9 @@
 						),
 					),
 				),
+				"section-title-links": (
+					"overflow-x": hidden,
```

Comment:
I don't think this is really solving the issue. It's just hiding links that flow off the page:

![Screenshot 2024-03-22 at 1 10 23 PM](https://github.com/WPMedia/arc-themes-blocks/assets/4759882/d2f9158d-69eb-451a-a6e2-2bbd6586ace4)
(The rest of the word Culture, News, Business, and the first part of Footer titles are all off the page).

We'd want the links themselves to break to a new line so they all remain visible.